apps/fabric-website/src/root.tsx

```diff
@@ -11,6 +11,8 @@ import { AndroidLogo, AppleLogo, WebLogo } from './utilities/index';
 
 initializeFileTypeIcons('https://static2.sharepointonline.com/files/fabric/assets/item-types-fluent/');
 
+setRTL(false);
```

Comment:
This really shouldn't be required. Did you debug the old version to find what codepath was setting RTL, and why it's no longer done?

Reply:
i dug around for a while but couldn't find anything. we aren't use setRTL anywhere in the website or example-app-base